Tests/Services/NfieldMeServiceTest.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Moq;
+using Newtonsoft.Json;
+using Nfield.Infrastructure;
+using Nfield.Models;
+using Nfield.Services.Implementation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Nfield.SDK.Services.Implementation;
+using Nfield.SDK.Models;
+using System.Net.Mail;
+using Nfield.SDK.Services;
+
+namespace Nfield.Services
+{
+    public class NFieldMeServiceTest : NfieldServiceTestsBase
+    {
+
+        [Fact]
+        public void Test_GetQuotaFrameVersionsAsync_SurveyIdIsNull_Throws()
+        {
+            var mockedNfieldConnection = new Mock<INfieldConnectionClient>();
+
+            var target = new NFieldMeService();
+            target.InitializeNfieldConnection(mockedNfieldConnection.Object);
+
+            Assert.Throws<ArgumentNullException>(() => UnwrapAggregateException(target.GetUserRole()));
+        }
+
+        [Fact]
+        public void TestGetRoles_Returns_User_Roles()
+        {
+            const HttpStatusCode httpStatusCode = HttpStatusCode.OK;
+
+            var permissions = new List<string>();
+            permissions.Add("Activity.Read");
+            permissions.Add("SamplingPoint.Read");
+            permissions.Add("InterviewQuality.Read");
```

Comment:
You can directly create and initialize the permissions
```suggestion
            var permissions = new List<string>{
               "Activity.Read",
               "SamplingPoint.Read",
               "InterviewQuality.Read"
            }
```
The same with the userRoles below

Reply:
> 

done